src/lib/eliom_reference.server.ml

```diff
@@ -29,16 +27,16 @@ module Ocsipersist = struct
   include Eliom_common.Ocsipersist.Polymorphic
 end
 
-let pers_ref_store = Ocsipersist.open_store "eliom__persistent_refs"
+let pers_ref_store = lazy (Ocsipersist.open_store "eliom__persistent_refs")
```

Comment:
?

Reply:
This was a `Lwt.t` before. This is an attempt to make sure that effects are caught by `Eio_main`.